_topic_maps/_topic_map.yml

```
@@ -119,6 +119,8 @@ Topics:
   File: getting-started-with-argo-rollouts
 - Name: Routing traffic by using Argo Rollouts
   File: routing-traffic-by-using-argo-rollouts
+- Name: Enable support for namespace-scoped Argo Rollouts installation
```

```suggestion
- Name: Enabling support for a namespace-scoped Argo Rollouts installation
```
From the [OCP documentation guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings): "For assemblies and task modules, use a gerund form in headings"

---

Fixed.